proto/cosmos/auth/v1beta1/auth.proto

```diff
@@ -36,11 +37,11 @@ message Params {
   option (gogoproto.equal)            = true;
   option (gogoproto.goproto_stringer) = false;
 
-  uint64 max_memo_characters     = 1 [(gogoproto.moretags) = "yaml:\"max_memo_characters\""];
-  uint64 tx_sig_limit            = 2 [(gogoproto.moretags) = "yaml:\"tx_sig_limit\""];
-  uint64 tx_size_cost_per_byte   = 3 [(gogoproto.moretags) = "yaml:\"tx_size_cost_per_byte\""];
-  uint64 sig_verify_cost_ed25519 = 4
+  google.protobuf.UInt64Value max_memo_characters     = 1 [(gogoproto.moretags) = "yaml:\"max_memo_characters\""];
```

Comment:
Why are these changes necessary?

Reply:
I thought since ParamSetPair's get stored individually, we need uint64 to be using `UIint64Value`. There could be another strategy where a given module's params proto message uses primitive types (not wrappers), and then we wrap them back into `UInt64Value` or `StringValue` wrapper types later in go code, but this seemed to be a bit uglier.

Reply:
Sorry I misread, but this makes sense!